index.bs

```diff
@@ -867,8 +863,8 @@ with the internal slots described in the following table:
         </tr>
         <tr>
             <td><dfn attribute for=Sensor>\[[waitingForUpdate]]</dfn></td>
-            <td>A boolean which indicates whether the observers have been updated
-                or whether the object is waiting for a new reading to do so.
+            <td>A boolean which indicates whether the observers need to be
```

Comment:
I think, semantics has changed a bit. It's not "waiting for update" flag, it is "Notify sensor object about new reading task is still running". Maybe we can rename this internal slot?